File: examples/helloworld/stream/node/streamsResponsesFromTheServerUsingWebsockets.js
const { HelloworldService } = require("m3o/helloworld");

// Stream returns a stream of "Hello $name" responses
async function streamsResponsesFromTheServerUsingWebsockets() {
  let helloworldService = new HelloworldService(process.env.MICRO_API_TOKEN);
  let rsp = await helloworldService.stream({
    messages: 10,
    name: "John",
  });
  console.log(rsp);
}

streamsResponsesFromTheServerUsingWebsockets();

File: examples/helloworld/stream/go/streamsResponsesFromTheServerUsingWebsockets.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/helloworld"
)

// Stream returns a stream of "Hello $name" responses
func StreamsResponsesFromTheServerUsingWebsockets() {
	helloworldService := helloworld.NewHelloworldService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := helloworldService.Stream(&helloworld.StreamRequest{
		Messages: 10,
		Name:     "John",
	})
	fmt.Println(rsp, err)
}
